indra/newview/skins/default/xui/en/menu_object.xml

```diff
@@ -183,6 +183,28 @@
     <menu_item_call.on_enable
         function="Tools.EnableTakeCopy" />
   </menu_item_call>
+  <menu_item_call
+      label="Take separate"
```

Comment:
This name is a bit confusing, needs a better name for taking objects individualy. Might even need to have 'Take' renamed to indicate that it takes whole selection as a single item.